detox/.detoxrc.json

```diff
@@ -20,7 +20,7 @@
     "android.emu.debug": {
       "type": "android.emulator",
       "binaryPath": "../android/app/build/outputs/apk/debug/app-debug.apk",
-      "build": "cd .. && ./node_modules/.bin/jetify && cd android && ./gradlew clean assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ../detox",
+      "build": "cd .. && ./node_modules/.bin/jetify && cd android && ./gradlew clean && ./gradlew assembleDebug assembleAndroidTest -DtestBuildType=debug && cd ../detox",
```

Comment:
Still include `clean` but on a separate command as suggested from an issue discussions.

Reply:
That make sense since individually clean and assembleDebug works